executor/src/witgen/generator.rs

```diff
@@ -73,10 +76,28 @@ impl<'a, T: FieldElement> Generator<'a, T> {
         Q: FnMut(&str) -> Option<T> + Send + Sync,
     {
         let first_row = self.compute_partial_first_row(mutable_state);
-        self.data = self.process(first_row, mutable_state);
+        self.data = self.process(vec![first_row], 0, mutable_state);
+        self.fill_remaining_rows(mutable_state);
         self.fix_first_row();
     }
 
+    fn fill_remaining_rows<Q>(&mut self, mutable_state: &mut MutableState<'a, T, Q>)
+    where
+        Q: FnMut(&str) -> Option<T> + Send + Sync,
+    {
+        if self.data.len() < self.fixed_data.degree as usize + 1 {
```

Comment:
For now, in practice, this condition will always be false. However, if we instantiated `Generator` with `instr_return` as latch, this would fill up the remaining rows. In the future, this will be needed to fill up rows for secondary VMs.